yoda/gas.go

```diff
@@ -80,11 +80,11 @@ func getRequestMsgByteLength(f FeeEstimationData) uint64 {
 	return size
 }
 
-func getReportMsgByteLength(msg sdk.Msg) uint64 {
+func getReportMsgByteLength(msg codec.ProtoMarshaler) uint64 {
 	return uint64(len(cdc.MustMarshalBinaryBare(msg)))
```

Comment:
Can cast to report msg and marshall it?

Reply:
I have fixed to `types.MsgReportData` because there is cast function at the `estimateGas()`